<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loggingWindow</class>
 <widget class="QWidget" name="loggingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1037</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPlainTextEdit" name="logTextDisplay">
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::ClickFocus</enum>
     </property>
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::LineWrapMode::NoWrap</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="annotateLabel">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Annotation:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="userAnnotationText">
       <property name="minimumSize">
        <size>
         <width>290</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>You may enter your own log notes here.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="annotateBtn">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Adds user-text to the log.</string>
       </property>
       <property name="text">
        <string>Annotate</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="debugBtn">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enable or disable debug logging. Use the &amp;quot;-d&amp;quot; or &amp;quot;--debug&amp;quot; flag to open wfview with debug logging enabled on startup. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Debug</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="commDebugChk">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This enables the logging of nearly all CI-V traffic. &lt;span style=&quot; font-weight:600;&quot;&gt;Use with caution&lt;/span&gt;. It is a lot of data. Meter levels and scope data are not shown. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>CommDebug</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="rigctlDebugChk">
       <property name="text">
        <string>RigCtl Debug</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="toBottomBtn">
       <property name="toolTip">
        <string>Scroll to bottom</string>
       </property>
       <property name="text">
        <string>Scroll Down</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearDisplayBtn">
       <property name="toolTip">
        <string>Clears the display. Does not clear the log file.</string>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openDirBtn">
       <property name="toolTip">
        <string>Makes a best-effort to ask the host system to open the log file directory.</string>
       </property>
       <property name="text">
        <string>Open Log Directory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openLogFileBtn">
       <property name="toolTip">
        <string>Makes a best-effort to ask the host system to open the logfile.</string>
       </property>
       <property name="text">
        <string>Open Log</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copyPathBtn">
       <property name="toolTip">
        <string>Copy the path of the log file to your clipboard.</string>
       </property>
       <property name="text">
        <string>Copy Path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sendToPasteBtn">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Sends text to termbin.com. Some personal information (such as your username) is in the log file, so do not click this button unless you are ok sharing your log file. This is a quick way to receive a URL, pointing to your log file text, that you can send to other people. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Send to termbin.com</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
